snowshu/core/replica/replica_factory.py

```diff
@@ -72,15 +71,15 @@ def _execute(self,
                     flags=re.IGNORECASE,
                 )
             )
-            logger.info(f"Incremental target catalog: {incremental_target_catalog}")
-
             apply_source_case = alter_relation_case(
                 case_function=self.config.source_profile.adapter._correct_case  # noqa pylint: disable=protected-access
             )
             incremental_target_catalog_casted = set(
                 map(apply_source_case, incremental_target_catalog)
             )
-
+            SnowShuGraph.catalog_difference(
```

Comment:
This seems to be executed twice, is that expected?

Reply:
No, I must copied it accidentally, thanks for notice